src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -165,6 +166,10 @@ class spell_hun_generic_scaling : public AuraScript
             SpellSchoolMask schoolMask = SpellSchoolMask(aurEff->GetSpellInfo()->Effects[aurEff->GetEffIndex()].MiscValue);
             int32 modifier = schoolMask == SPELL_SCHOOL_MASK_NORMAL ? 35 : 40;
             amount = CalculatePct(std::max<int32>(0, owner->GetResistance(schoolMask)), modifier);
+            if (owner->HasAura(SPELL_HUNTER_PET_LEGGINGS_OF_BEAST_MASTERY) && schoolMask == SPELL_SCHOOL_MASK_NORMAL)
+            {
+                amount += 490;
```

Comment:
famous question, how did we come up with this

Reply:
https://www.wowhead.com/wotlk/item=31328/leggings-of-beast-mastery

Reply:
Love you

Reply:
As much as I want to conjure random numbers, you won't let me 😠

Reply:
But you love me anyway, dont you :D

Reply:
https://www.youtube.com/watch?v=9wtvXoXh0VU

Reply:
Is it not part of the spell attribute? Should use spell/aura attribute instead of hardcoded number

Reply:
I could, however do it for https://github.com/azerothcore/azerothcore-wotlk/pull/17427